### PR TITLE
Add Nextcloud Deck Recipe

### PR DESCRIPTION
**Pull request by mindcombatant**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [ ] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md) for this project.
- [✔️] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.

#### Description of Change

Added Nextcloud Deck Recipe using other Nextcloud Recipes as a base. Icon was edited by myself. See recipes/nextcloud-deck/README.md for further details.

### CODE REVIEW

File: recipes/nextcloud-deck/package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "id": "nextcloud-deck",
+  "name": "Nextcloud Deck",
+  "version": "1.2.1",
```
Comment (vraravam) [inline recipes/nextcloud-deck/package.json:4]: when introducing a new recipe, we like to start with version as `1.0.0`

**Review by vraravam — CHANGES_REQUESTED** [review]

**Review by vraravam — APPROVED** [review]

File: recipes/nextcloud-deck/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "id": "nextcloud-deck",
+  "name": "Nextcloud Deck",
+  "version": "1.0.0",
+  "license": "MIT",
+  "repository": "https://github.com/mindcombatant/ferdium-recipes/tree/master/recipes/nextcloud-deck/",
+  "config": {
+    "hasNotificationSound": true,
+    "hasTeamId": true,
+    "urlInputPrefix": "https://",
+    "urlInputSuffix": "/apps/news/",
```
Comment (Alphrag) [inline recipes/nextcloud-deck/package.json:11]: The `serviceURL` below uses `/apps/deck`, but the user will see `/apps/news` as can be seen here:
Capture d’écran 2023-05-15 à 00 21 06
Please make this consistent.

**Review by Alphrag — CHANGES_REQUESTED** [review]

File: recipes/nextcloud-deck/package.json (hunk 2 above)
Comment (mindcombatant) [reply recipes/nextcloud-deck/package.json]: oh, whoops, i could've sworn i got all of them, thanks for pointing this out! should be fixed now.

**Review by Alphrag — APPROVED** [review]

**Comment by Alphrag** [comment]
@all-contributors please add @mindcombatant for code

**Pull request merged**